app/Nova/Resources/Integration.php

```diff
@@ -67,29 +68,37 @@ public function fields(NovaRequest $request): array
 
         return [
             ID::make()
-                ->readonly(),
+                ->readonly()
+                ->onlyOnDetail(),
+
+            DateTime::make('Date created', 'created_at')
+                ->readonly()
+                ->exceptOnForms()
+                ->filterable()
+                ->sortable()
+                ->displayUsing(fn ($date) => $date->format('d/m/Y')),
+
+            Text::make('Name')
```

Comment:
As posted in Slack I prefer the name as the first column, now that the ID is gone.